docs/community_leadership/community_leadership.rst

```diff
@@ -134,57 +134,26 @@ The current leadership team and their terms are as follows:
      - 2024
      - 2027
    * - Sven Döring
-     - Marketing Team Assistant Lead
-     - sven.doering@mautic.org
-     - 2023
-     - 2025
-   * - Mike Van Hemelrijck
      - Assistant Team Lead, Marketing
-     - mike.hemelrijck@mautic.org
-     - 2024
-     - 2027
-   * - Norman Pracht
-     - Product Team Assistant Lead (Development)
-     - norman.pracht@mautic.org
+     - sven.doering@mautic.org
      - 2023
      - 2025
```

Comment:
Do we need to change the last year for Sven here?

Reply:
Hmm, I'm not sure - I think it should stay as it is but he's the team lead now.

Reply:
I'll fix the title to team lead, then.